cli/login.go

```diff
@@ -17,24 +18,40 @@ import (
 	"github.com/getoutreach/gobox/pkg/box"
 )
 
-// EnsureLoggedIn ensures that we are authenticated with Vault and have a valid token
-func EnsureLoggedIn(ctx context.Context, log logrus.FieldLogger, b *box.Config) ([]byte, error) {
+// EnsureLoggedIn ensures that we are authenticated with Vault and have a valid token,
+// returning the token and expiration date.
+func EnsureLoggedIn(ctx context.Context, log logrus.FieldLogger, b *box.Config, minTimeRemaining time.Duration) ([]byte, time.Time, error) {
 	// Check if we need to issue a new token
-	token, err := IsLoggedIn(ctx, log, b)
+	refresh := false
```

Comment:
```suggestion
	var refresh bool
```
if you want idiomatic go suggestions

> Make the zero value useful.
> - https://go-proverbs.github.io/

Reply:
i think that's uglier but w/e